### PR TITLE
fixes #121 - Remove libskycoin path from settings

### LINKED ISSUE #121: Remove LIBSKYCOIN_PATH from settings
Should not be needed now that `PySkycoin` is a project dependency .

### PR DESCRIPTION
**Pull request by stdevOzkar**
Remove LIBSKYCOIN_PATH from `settings.py` refs #121 close #121

### CODE REVIEW

File: exchange_app/api_1_0/transactions.py [hunk 1]
```diff
@@ -1,38 +1,65 @@
...
-    if not request.json:
-        return make_response(jsonify(build_error("Input format error")), 400)
-    params = {'operationID', 'inputs', 'toAddress', 'assetId'}
-    if all(x not in params for x in request.json):
-        return make_response(jsonify(build_error("Input data error")), 400)
-    result = transaction_many_inputs(request.json)
-    if "transactionContext" in result:
-        return jsonify(result)
-    return jsonify({"status": 500, "error": "Invalid response"})
+@api.route('/transactions/unconfirmed', methods=['GET'])
```
Comment (olemis) [inline exchange_app/api_1_0/transactions.py:10]: how is this change related to `LIBSKYCOIN_PATH` ?

**Review by olemis — CHANGES_REQUESTED** [review]
In many places I see changes not related to removing `LIBSKYCOPIN_PATH` . Is this a merging issue?

File: exchange_app/api_1_0/blockchain.py [hunk 2]
```diff
@@ -253,50 +288,82 @@ def get_address_transactions(address):
...
+        )
 
     retvalue = {
         "signedTransaction": signedHashHex
     }
     return jsonify(retvalue)
 
 
-def transaction_many_inputs(values):
+def get_unconfirmed_txs():
```
Comment (olemis) [inline exchange_app/api_1_0/blockchain.py:360]: This endpoint does not exist any more. Deprecated .

**Review by olemis — CHANGES_REQUESTED** [review]

**Comment by olemis** [comment]
@stdevOzkar issue overflow with this PR . I'd suggest you to retry this using current code in `develop` branch.

**Comment by stdevOzkar** [comment]
@olemis huge mess is this PR. I've used my local develop branch as base branch. Fixing in a jiffy.
